getArtifactUrl.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
remove.

in future we can use their API to get the assigned domain name and post it to the pr